common/persistence/sql/sqlplugin/mysql/visibility.go

```diff
@@ -42,7 +42,7 @@ const (
 	templateCreateWorkflowExecutionClosed = `INSERT INTO executions_visibility (` +
 		`namespace_id, workflow_id, run_id, start_time, execution_time, workflow_type_name, close_time, status, history_length, memo, encoding) ` +
 		`VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) ` +
-		`ON DUPLICATE KEY UPDATE start_time = VALUES(start_time), execution_time = VALUES(execution_time), workflow_type_name = VALUES(workflow_type_name), ` +
+		`ON DUPLICATE KEY UPDATE workflow_id = VALUES(workflow_id), start_time = VALUES(start_time), execution_time = VALUES(execution_time), workflow_type_name = VALUES(workflow_type_name), ` +
```

Comment:
This should not be a case, right? If `run_id` is the same `workflow_id` can't be changed. Actually surprised that it is not part of a key.

Reply:
the behavior on cassandra & postgresql are the same, basically overwrite
here (mysql) the change is not really necessary since the mapping from runID -> workflowID will not change (runID collision?)

i am making this change just to make the goal / behavior the same on all 3 DBs